package.json

```
@@ -121,5 +121,8 @@
     "webpack-cli": "^4.6.0",
     "webpack-dev-server": "^4.11.1",
     "webpack-merge": "^5.1.4"
+  },
+  "dependencies": {
+    "moment": "^2.29.4"
```

Why is this being added?
```suggestion
  },
  "dependencies": {
    "moment": "^2.29.4"
```